projects/angular/src/forms/styles/_properties.forms.scss

```diff
@@ -96,6 +96,26 @@
       // Range
       --clr-forms-range-progress-fill-color: var(--clr-color-action-600);
       --clr-forms-range-track-color: var(--clr-color-neutral-200);
+
+      // Password
+      --clr-forms-password-input-height: #{forms-variables.$clr-forms-password-input-height};
+      --clr-forms-password-padding-bottom: #{forms-variables.$clr-forms-password-padding-bottom};
+      --clr-forms-password-layout-space: #{forms-variables.$clr-forms-password-layout-space};
+      --clr-forms-password-eye-icon-min-height: #{forms-variables.$clr-forms-password-eye-icon-min-height};
+      --clr-forms-password-eye-icon-width: #{forms-variables.$clr-forms-password-eye-icon-width};
```

Comment:
I would combine these two in one `--clr-forms-password-eye-icon-size` as it will be consistent will the others and it will use our mixing for that `equilateral`. In my datalist [PR](https://github.com/vmware-clarity/ng-clarity/pull/795) I created a mixin which provides fallback functionality.

Reply:
will check it, thanks